# MongoDB keeps track of its build parameters, which tricks nix into
# keeping dependencies to build inputs in the final output.
# We remove the build flags from buildInfo data.
--- a/site_scons/mongo/generators.py
+++ b/site_scons/mongo/generators.py
@@ -18,10 +18,7 @@ def default_buildinfo_environment_data():
         ('distmod', '$MONGO_DISTMOD', True, True,),
         ('distarch', '$MONGO_DISTARCH', True, True,),
         ('cc', '$CC_VERSION', True, False,),
-        ('ccflags', '$CCFLAGS', True, False,),
         ('cxx', '$CXX_VERSION', True, False,),
-        ('cxxflags', '$CXXFLAGS', True, False,),
-        ('linkflags', '$LINKFLAGS', True, False,),
         ('target_arch', '$TARGET_ARCH', True, True,),
         ('target_os', '$TARGET_OS', True, False,),
     )
